Runtime/AvatarCreator/Scripts/UI/Elements/LoginElement.cs

```diff
@@ -32,12 +34,18 @@ private void OnDisable()
             AuthManager.OnSignInError -= LoginFailed;
         }
 
+        private void OnDestroy()
+        {
+            cancellationTokenSource.Cancel();
+            cancellationTokenSource.Dispose();
+        }
+
         /// <summary>
         /// Sends a verification code to the email address that was entered into the email InputField.
         /// </summary>
-        public void SendVerificationCode()
+        public async void SendVerificationCode()
         {
-            AuthManager.SendEmailCode(emailField.text);
+            await TaskExtensions.HandleCancellation(AuthManager.SendEmailCode(emailField.text, cancellationTokenSource.Token));
```

Comment:
I appreciate that you want to address all the asnyc events/requests but think this cancellation token is overkill. If this function did something destructive or that changed the state and took a significant time then it might make sense but its super fast whether this runs or not it doesn't really effect much.

Reply:
There's nothing in the "Super fast" with network connections. I wanted to cover everything so that there are no issues with it. I understand, that it might be overkill, but still, it's a place, that might have a point of failure.